Sources/TuistGraph/Models/BuildRule+CompilerSpec.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+
+extension BuildRule {
+    public enum CompilerSpec: String, Codable {
+        case appIntentsMetadataExtractor = "com.apple.compilers.appintentsmetadata"
```

Comment:
Maybe we can document here how we got the values. It could be valuable to have this information in code in case we need to update the cases in the future.

Reply:
ok, done 🙂